client/question/components/LearnerViewDirective.js

```diff
@@ -70,23 +70,29 @@ tie.directive('learnerView', [function() {
             </div>
             <div class="tie-coding-ui">
               <div class="tie-lang-terminal">
-                <div class="tie-coding-terminal">
-                  <div class="tie-codemirror-container"
-                      tabindex="0"
-                      ng-keypress="onKeypressCodemirrorContainer($event)"
-                      ng-focus="onFocusCodemirrorContainer()">
-                    <ui-codemirror ui-codemirror-opts="codeMirrorOptions"
-                        ng-model="editorContents.code"
-                        ng-change="onCodeChange()"
-                        ng-if="!accessibleMode"
-                        class="protractor-test-code-input">
-                    </ui-codemirror>
-                    <ui-codemirror ng-model="editorContents.code"
-                        ui-codemirror-opts="accessibleCodeMirrorOptions"
-                        ng-change="onCodeChange()"
-                        ng-if="accessibleMode"
-                        class="protractor-test-code-input">
-                    </ui-codemirror>
+                <div class="tie-user-terminal">
+                  <div class="tie-coding-terminal">
+                    <div class="tie-codemirror-container"
+                        tabindex="0"
+                        ng-keypress="onKeypressCodemirrorContainer($event)"
+                        ng-focus="onFocusCodemirrorContainer()">
+                      <ui-codemirror ui-codemirror-opts="codeMirrorOptions"
+                          ng-model="editorContents.code"
+                          ng-change="onCodeChange()"
+                          ng-if="!accessibleMode"
+                          class="protractor-test-code-input">
+                      </ui-codemirror>
+                      <ui-codemirror ng-model="editorContents.code"
+                          ui-codemirror-opts="accessibleCodeMirrorOptions"
+                          ng-change="onCodeChange()"
+                          ng-if="accessibleMode"
+                          class="protractor-test-code-input">
+                      </ui-codemirror>
+                    </div>
+                  </div>
+                  <div class="tie-print-terminal">
```

Comment:
I thought we were doing this with a flag so that we can tweak the "display mode" as needed? I.e. by changing a constant in the frontend config we can control whether the print terminal is shown or not?